### DISCUSSION TITLE
v1.0.0-rc.1

### DISCUSSION THREAD

**Discussion by ktmitton** [ktmitton/Mittons.Fixtures#64] [Announcements]
This is the first release candidate for this package before the 1.0 release. The minimum requirements were to implement an Sftp server that could be stood up for testing, and allow files to be added or removed as needed.

<hr /><em>This discussion was created from the release <a href='https://github.com/ktmitton/Mittons.Fixtures/releases/tag/v1.0.0-rc.1'>v1.0.0-rc.1</a>.</em>

**Comment by Frueber** [comment]
Just noting on this announcement that rc-1 directly depends on the Xunit test framework. That's not a stopper, but this is planned to be removed in an upcoming release candidate.

**Reply by ktmitton** [reply]
Yup, rc 2 will get that removed! It's definitely a stopper for actual release, as this changes the public interface, so if we didn't include it in 1.0, the release that includes it would need to be a major jump to 2.0, and I don't want to do that this early in the project.
